### PR TITLE
nav completed

### PR DESCRIPTION
**Pull request by TahaNamdar**

### CODE REVIEW

File: src/components/menu/styles/Navbar.styles.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { NavbarProps } from '../content/navbar_types';
+export const Nav = styled.div<NavbarProps>(({ theme }) => ({
```
Comment (alirezatahani0) [inline src/components/menu/styles/Navbar.styles.ts:3]: az HTML tag nav estefade kon

File: src/components/menu/content/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { NavbarProps } from './navbar_types';
+import { Row, Col } from '../../grid/Index';
```
Comment (alirezatahani0) [inline src/components/menu/content/Navbar.tsx:3]: pull develop and change the import

File: src/components/menu/content/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import {
+  Nav,
+  Item,
+  Ul,
+  Logo,
+  SocialItem,
+  SocialContent,
+} from '../styles/Navbar.styles';
+import { FaAtlassian, FaFacebook, FaTwitter } from 'react-icons/fa';
```
Comment (alirezatahani0) [inline src/components/menu/content/Navbar.tsx:12]: need empty line after imports

File: src/components/menu/content/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+const Navbar: React.FC<NavbarProps> = ({ ...rest }: NavbarProps) => {
+  return (
+    <Nav>
+      <Row>
+        <Col md={4}>
+          <Logo>
+            <FaAtlassian size={18} />
+          </Logo>
+        </Col>
+        <Col md={4}>
```
Comment (alirezatahani0) [inline src/components/menu/content/Navbar.tsx:22]: routing Column should be bigger than the others

File: src/components/menu/content/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          <Ul>
+            <Item>Home</Item>
```
Comment (alirezatahani0) [inline src/components/menu/content/Navbar.tsx:24]: add Menu object under utils/constants.ts and do a loop here

File: src/components/menu/styles/Navbar.styles.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { NavbarProps } from '../content/navbar_types';
```
Comment (alirezatahani0) [inline src/components/menu/styles/Navbar.styles.ts:2]: need empty line after imports

File: src/components/menu/styles/Navbar.styles.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  alignItems: 'center',
+}));
+export const SocialContent = styled.div<NavbarProps>(({ theme }) => ({
+  textAlign: 'center',
+  marginLeft: 17,
+  ...theme.typography.body1,
+}));
+export const Logo = styled.div<NavbarProps>(({ theme }) => ({
+  textAlign: 'center',
+  marginLeft: 24,
```
Comment (alirezatahani0) [inline src/components/menu/styles/Navbar.styles.ts:27]: why marginLeft??

**Review by alirezatahani0 — CHANGES_REQUESTED** [review]

File: src/components/menu/styles/Navbar.styles.ts (hunk 7 above)
Comment (TahaNamdar) [reply src/components/menu/styles/Navbar.styles.ts:27]: az ham item ha fasele begire

**Pull request closed without merging**